cratedb_sqlparse_py/tests/test_enricher.py

```diff
@@ -0,0 +1,48 @@
+from cratedb_sqlparse.parser import Metadata
+
+
+def test_table_metadata():
+    from cratedb_sqlparse import sqlparse
+
+    query = "SELECT 1; SELECT 2;"
+    stmts = sqlparse(query)
+    for stmt in stmts:
+        assert hasattr(stmt, "metadata")
+        assert isinstance(stmt.metadata, Metadata)
```

Comment:
This suggestion will make the error on CI go away.
```python
ModuleNotFoundError: No module named 'cratedb_sqlparse.generated_parser.SqlBaseParser'
```
-- https://github.com/crate/cratedb-sqlparse/actions/runs/9467617375/job/26082048675?pr=36#step:6:59

```suggestion
def test_table_metadata():
    from cratedb_sqlparse import sqlparse
    from cratedb_sqlparse.parser import Metadata

    query = "SELECT 1; SELECT 2;"
    stmts = sqlparse(query)
    for stmt in stmts:
        assert hasattr(stmt, "metadata")
        assert isinstance(stmt.metadata, Metadata)
```

Reply:
The reason is because on an uninitialized project, pytests code discovery will import all modules. This will croak when runtime grammar has not been generated yet.

It is currently happening too late, on behalf of `conftest.py`. It would need to be invoked before the discovery process, so we can have the module imports on module scope again. Currently, it is not, so module imports need to be inlined into functions.